generator/lookml.py

```diff
@@ -118,7 +121,29 @@ def _glean_apps_to_v1_map(glean_apps):
     return {d["name"]: d["v1_name"] for d in glean_apps}
 
 
-def _lookml(namespaces, glean_apps, target_dir, dryrun, namespace_filter=[]):
+def _run_generation(func):
+    """
+    Run the partially applied generate function.
+
+    For parallel execution.
+    """
+    return func()
```

Comment:
I'd think there should be a more straightforward way to execute a bunch of ready-to-run function calls in parallel without this kind of shim, but I'm not seeing such a way. 🤷 

I guess the alternative in this case would be to build up lists of arguments to the generation functions and then parallel-map those generation functions to the arguments, but then you'd lose the flexibility to do things like combining the view and datagroup generation into one set.